### PR TITLE
Storage RRD - descriptionlength fix - percent column fix

### PR DESCRIPTION
**Pull request by SourceDoctor**
increase title description from 12 to 16 characters
also change columntitle vom "%age" to "Ratio" for percental usage column

### CODE REVIEW

File: includes/html/graphs/device/storage.inc.php [hunk 1]
```diff
@@ -27,7 +27,7 @@
         $iter   = '0';
     }
 
-    $descr        = rrdtool_escape($storage['storage_descr'], 12);
+    $descr        = rrdtool_escape($storage['storage_descr'], 16);
```
Comment (Jellyfrog) [inline includes/html/graphs/device/storage.inc.php]: Other graphs also seems to have 12 chars, like: https://github.com/librenms/librenms/blob/master/includes/html/graphs/storage/usage.inc.php#L19  
If we change one, we should change all?

File: includes/html/graphs/device/storage.inc.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 require 'includes/html/graphs/common.inc.php';
 
 $iter         = '1';
-$rrd_options .= " COMMENT:'                    Size      Used    %age\\l'";
+$rrd_options .= " COMMENT:'                        Size      Used    Ratio\\l'";
```
Comment (Jellyfrog) [inline includes/html/graphs/device/storage.inc.php]: Shouldn't it be "% Used", like the other graphs?

Comment (SourceDoctor) [reply includes/html/graphs/device/storage.inc.php]: @Jellyfrog i changed it to "% Used"

File: includes/html/graphs/device/storage.inc.php (hunk 1 above)
Comment (SourceDoctor) [reply includes/html/graphs/device/storage.inc.php]: @Jellyfrog 
i agree.
is also changed now.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-56-release-changelog-september-2019/9671/1

**Pull request merged**